source/_docs/http-to-https.md

```diff
@@ -117,9 +117,11 @@ At this point, all visitors to the site should be able to securely access all pa
 
 ## Redirect to HTTPS and the primary domain
 
-As part of best security practices, we suggest you [Require HTTPS with the HSTS Header](/docs/hsts), as well as a redirect.
+As part of best security practices, we suggest you [Require HTTPS with the HSTS Header](/docs/hsts/), as well as redirect all HTTP traffic to HTTPS.
```

Comment:
Wordsmithing: Maybe this should say "which has the effect of redirecting all HTTP traffic to HTTPS", as it is not a separate operation to redirect HTTP traffic to HTTPS but a consequence of setting the HSTS header.